sync/statesync/code_queue.go

```diff
@@ -41,33 +42,30 @@ type CodeQueue struct {
 	closeChanOnce sync.Once // See usage in [CodeQueue.closeOutChannelOnce]
 }
 
-// TODO: this will be migrated to using libevm's options pattern in a follow-up PR.
-type codeQueueOptions struct {
+type queueConfig struct {
```

Comment:
This feels like overkill to just specify a capacity. Why can't it be passed directly?